public/doc/openapi-3-v1.0.0.json

```diff
@@ -4067,6 +4067,31 @@
           }
         }
       },
+      "Tenant": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
```

Comment:
Should this be `"$ref": "#/components/schemas/ID"` instead?

Reply:
:+1: done